src/GitHub.TeamFoundation.14/Home/GitHubHomeSection.cs

```diff
@@ -175,8 +170,28 @@ void ShowWelcomeMessage()
                 }),
                 false,
                 welcomeMessageGuid);
+
+            settings.WelcomeMessageLastSeen = DateTime.UtcNow;
+            settings.Save();
         }
 
+        //todo:temporary
+        static int GetIso8601WeekOfYear(DateTimeOffset time)
```

Comment:
Yeah, we should probably move this into a utils class somewhere as it's used in two places now.